### PR TITLE
Adding starkbank-ecdsa

### PR DESCRIPTION
**Pull request by cjmartian**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/starkbank-ecdsa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - ellipticcurve
+    - ellipticcurve.utils
+
+about:
+  home: "https://github.com/starkbank/ecdsa-python.git"
```
Comment (dopplershift) [inline recipes/starkbank-ecdsa/meta.yaml:30]: This URL should not have the `.git`

File: recipes/starkbank-ecdsa/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "A lightweight and fast pure python ECDSA library"
+  doc_url: 
+  dev_url:
```
Comment (dopplershift) [inline recipes/starkbank-ecdsa/meta.yaml:36]: Should either point to the GitHub URL above or be removed.

File: recipes/starkbank-ecdsa/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    - ellipticcurve
+    - ellipticcurve.utils
+
+about:
+  home: "https://github.com/starkbank/ecdsa-python.git"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "A lightweight and fast pure python ECDSA library"
+  doc_url:
```
Comment (dopplershift) [inline recipes/starkbank-ecdsa/meta.yaml:35]: If there's not a sensible URL to use for docs should be removed.

**Review by dopplershift — CHANGES_REQUESTED** [review]

**Comment by cjmartian** [comment]
@dopplershift Thanks for the review, I made the changes you requested.

**Comment by dopplershift** [comment]
Merging since noarch:python and worked on linux.

**Pull request merged**